app/controllers/users.js

```diff
@@ -12,19 +12,43 @@ const encryptPassword = ({ password }) => {
   return bcrypt.hash(password, salt);
 };
 
+const sendUser = (usr, res, message) => {
+  const User = userSerializer(usr);
+  logger.info(message);
+  return res.status(201).send(User);
+};
+
+const createUser = (usr, isAdmin = false) => {
+  const User = userCreationSerializer(usr);
+  return encryptPassword(User).then(hashedPassword => {
+    const fields = { ...User, password: hashedPassword, isAdmin };
+    return userService.create(fields);
+  });
+};
+
 exports.create = (req, res, next) =>
   user
     .findOne({ where: { email: req.body.email } })
     .then(userInstance => {
-      if (userInstance) throw next(errors.emailAlreadyExists('The email already exists'));
-      const usr = userCreationSerializer(req.body);
-      return encryptPassword(usr).then(hashedPassword => {
-        const fields = { ...usr, password: hashedPassword };
-        return userService.create(fields).then(newUser => {
-          const User = userSerializer(newUser);
-          logger.info(`The user ${JSON.stringify(User)} was successfully created`);
-          res.status(201).send(User);
+      if (userInstance) throw errors.emailAlreadyExists('The email already exists');
+      createUser(req.body).then(newUser => {
+        sendUser(newUser, res, 'The user was successfully created');
+      });
```

Comment:
soory, i forgot this.  You can do this `createUser(req.body).then(newUser => sendUser(newUser, res, 'The user was successfully created'));`

Reply:
And add a return before createUser